packages/BiostructureViewer/src/utils/pdb-grid-cell-renderer.ts

```diff
@@ -120,15 +120,20 @@ export class PdbGridCellRenderer extends DG.GridCellRenderer {
             height: gridCell.grid.props.rowHeight * r,
           },
           onAfterRender: async (canvas: HTMLCanvasElement) => {
-            _package.logger.debug('PdbRenderer.render() onAfterRender() ' + `rowIdx = ${rowIdx}`);
-            const imageStr: string = canvas.toDataURL();
-            const image: HTMLImageElement = await base64ToImg(imageStr);
-
-            const imageCache = gridCell.tableColumn!.temp.get(PDB_RENDERER_IMAGE_CACHE_KEY);
-            imageCache[rowIdx] = image;
-            gridCell.tableColumn!.temp.set(PDB_RENDERER_IMAGE_CACHE_KEY, imageCache);
-
-            service.renderOnGridCell(g, new DG.Rect(x, y, w, h), gridCell, image);
+            g.save();
+            try {
+              _package.logger.debug('PdbRenderer.render() onAfterRender() ' + `rowIdx = ${rowIdx}`);
+              const imageStr: string = canvas.toDataURL();
```

Comment:
This makes no sense to me. We have two canvases, we want to render the content of canvas 1 on top of canvas 2. Why are we creating a string for it, can't we do it more efficiently?

Reply:
This was for debugging purposes. I removed these debug remnants.